bedrock/externalpages/templates/externalpages/pocket/add.html

```diff
@@ -0,0 +1,116 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% block page_title %} How to Save {% endblock %}
+
+{% block pocket_css %}
+  {{ css_bundle('pocket-add') }}
+{% endblock  %}
+
+{% block content %}
+
+{% include 'externalpages/pocket/includes/platform-nav.html' %}
+
+<div class="add-page">
+   <section class="add-header">
+      <div class="add-header__wrapper">
+         <div class="add-header__title">
+            <h2>Save to Pocket from Firefox</h2>
+         </div>
+         <div class="add-header__context">
+            <div class="add-header__context-image"><img src="{{ static('img/externalpages/pocket/add-pocket-header.jpg') }}" alt="Pocket logo in browser taskbar"></div>
+            <div class="add-header__context-info">
+               <p>
+                  Click the Pocket button in your Firefox toolbar to save articles, videos,
+                  pages, and more.
+               </p>
+               <a href="https://help.getpocket.com/article/900-saving-to-pocket-and-viewing-your-list-in-firefox" class="btn red-btn" target="_blank" rel="noopener noreferrer">Learn How</a>
+               <p>
+                  <a
+                     href="https://help.getpocket.com/article/942-where-is-the-pocket-button-in-firefox"
+                     class="dark-inline-link" target="_blank" rel="noopener noreferrer">
+                  Don't see the Pocket button?
+                  <span class="rarrow">›</span>
```

Comment:
Not something that needs fixing now, but it would be better to move characters like this to pseudo elements in CSS to improve accessibility.